packages/xchain-radix/src/client.ts

```diff
@@ -0,0 +1,482 @@
+import { CommittedTransactionInfo, GatewayApiClient } from '@radixdlt/babylon-gateway-api-sdk'
+import {
+  LTSRadixEngineToolkit,
+  NetworkId,
+  PrivateKey,
+  RadixEngineToolkit,
+  SimpleTransactionBuilder,
+} from '@radixdlt/radix-engine-toolkit'
+import {
+  AssetInfo,
+  Balance,
+  BaseXChainClient,
+  FeeOption,
+  FeeType,
+  Fees,
+  Network,
+  PreparedTx,
+  Tx,
+  TxHistoryParams,
+  TxParams,
+  TxType,
+  TxsPage,
+  XChainClientParams,
+} from '@xchainjs/xchain-client'
+import { getSeed } from '@xchainjs/xchain-crypto/lib'
+import { Address, Asset, assetAmount, assetToBase, baseAmount } from '@xchainjs/xchain-util'
+import axios from 'axios'
+import {
+  AssetXRD,
+  KeyType,
+  MAINNET_GATEWAY_URL,
+  RadixChain,
+  STATE_ENTITY_DETAILS_PATH,
+  STOKENET_GATEWAY_URL,
+  TRANSACTION_COMMITTED_DETAILS_PATH,
+  TRANSACTION_CONSTRUCTION_PATH,
+  TRANSACTION_PREVIEW_PATH,
+  TRANSACTION_SUBMIT_PATH,
+  TransferTransactionManifest,
+  XRD_DECIMAL,
+} from './const'
+import { EntityDetailsResponse, Transaction } from './types/radix'
+
+/**
+ * Custom Radix client
+ */
+
+class Client extends BaseXChainClient {
+  privateKey: PrivateKey
+  gatewayApiClient: GatewayApiClient
```

Comment:
Similar thing happens with the network. If somebody calls `radixClient.setNetwork` the gateway client is not updated and continues to use the old network.

Reply:
@balda-rdx Ping on the above since its still true

Reply:
@balda-rdx Ping on this